src/main/java/org/jfrog/hudson/pipeline/declarative/steps/npm/NpmInstallStep.java

```diff
@@ -63,6 +64,11 @@ public void setResolverId(String resolverId) {
         this.resolverId = resolverId;
     }
 
+    @DataBoundSetter
+    public void setJavaArgs(String javaArgs) {
```

Comment:
I suggest adding the following comment:
This setter has been added to allow java remote debugging.

Reply:
I added the comment in the javaArgs variable above